Jenkinsfile

```
@@ -12,7 +12,7 @@ node {
       updateGithubCommitStatus('PENDING', "${env.WORKSPACE}/src")
 
       stage('Build debian package') {
-	sh 'make jammy_package'
+	sh 'make jammy_package EXOSCALE_DOCKER_REGISTRY=${EXOSCALE_DOCKER_REGISTRY}'
```

```suggestion
	sh 'make jammy_package EXOSCALE_DOCKER_REGISTRY=${EXOSCALE_DOCKER_REGISTRY}'
```

Shouldn't be prepended?

---

I think you're right; but somehow it worked here: https://ci.internal.exoscale.ch/blue/rest/organizations/jenkins/pipelines/puppet-omnibus/runs/6/nodes/17/steps/18/log/?start=0

---

Agree with @xgattuso ; if it works, it's confusing :wink:

---

Well, order doesn't play a role it seems.

```
➜  russell src/pkg/puppet-omnibus git:(fix_docker_registry) ✗ cat Makefile
.PHONY: my_target

my_target: $(info $(MYVAR))
        @:

➜  russell src/pkg/puppet-omnibus git:(fix_docker_registry) ✗ make MYVAR=la my_target 
la
➜  russell src/pkg/puppet-omnibus git:(fix_docker_registry) ✗ make my_target MYVAR=ko
ko
```

Will change it as you suggested, for the sake of clarity.

---

https://github.com/exoscale/puppet-omnibus/pull/2/commits/d864ed7a7e49bae00ce32dcb1d2b3cec07ebb324